lightwood/encoders/time_series/rnn.py

```diff
@@ -42,8 +42,15 @@ def setup_nn(self, additional_targets=None):
 
         total_dims = self._n_dims
         if additional_targets:
-            for _ in additional_targets:
-                total_dims += 1  # We are assuming target dim == 1
+            for t in additional_targets:
+                if t['original_type'] == 'categorical':
+                    t['normalizer'] = CatNormalizer()
+                    t['normalizer'].prepare(t['data'])
+                    total_dims += len(t['normalizer'].scaler.categories_[0])
```

Comment:
Is this adding 1 dimension per category basically ? If so I feel like it could get way too compute heavy very quickly, but for now I guess it works.

Reply:
This extends the input tensor dimension by the # of categories observed at train time. The encoder should stay at manageable computing requirements.